DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/Helpers/ParameterValueExtractor.cs

```diff
@@ -98,14 +107,20 @@ or _
     return GetValueGeneric(parameter, StorageType.ElementId, (parameter) => parameter.AsElementId());
   }
 
-  public T GetValueAsDocumentObject<T>(Element element, BuiltInParameter builtInParameter)
+  public T? GetValueAsDocumentObjectOrNull<T>(Element element, BuiltInParameter builtInParameter)
     where T : class
   {
     ElementId? elementId = GetValueAsElementId(element, builtInParameter);
     Element paramElement = element.Document.GetElement(elementId);
-    return paramElement as T
+    return paramElement as T;
+  }
```

Comment:
We should consider making this a `TryGet` pattern instead, as this doesn't make the distinction between a value that was in fact null, and a value that failed to cast to `T` but was not null.

```csharp
public bool TryGetValueAsDocumentObject<T>(
    Element element, 
    BuiltInParameter builtInParameter, 
    out T value
)
```